cmd/status/utils.go

```diff
@@ -87,6 +87,10 @@ func testExportedAPI(t *testing.T, done chan struct{}) {
 			"pause node",
 			testStopResumeNode,
 		},
+		{
+			"swarm upload/download file",
+			testSwarmUploadFile,
+		},
 	}
```

Comment:
- Unnecessary, duplicate test
- What is being tested here that hasn't (or couldn't have) been tested in `swarm_test.go`?
- The `cmd/status/utils.go` is for tests in **exposed** methods i.e. methods that we expose in `library.go` - you haven't exposed any methods, no need to update `cmd/status/*` tests